package.json

```diff
@@ -74,6 +74,7 @@
 		"eslint": "^8.6.0",
 		"eslint-ava-rule-tester": "^4.0.0",
 		"eslint-plugin-eslint-plugin": "^4.1.0",
+		"eslint-plugin-internal-rules": "file:./scripts/internal-rules/",
```

Comment:
Does eslint not offer a way to load a plugin from a file instead of a module?

Reply:
This being a loooong story, and the reason is ridiculous, add a plugin path in config, ESLint can't get a plugin name to use...

https://github.com/eslint/eslint/issues/3458

Unless you use the ESLint API, you can't add plugin in other way.

- https://github.com/sindresorhus/eslint-plugin-unicorn/blob/e8ab1766f367311b906d198021353c2bbf8df301/test/integration/package.json#L8
- https://github.com/prettier/eslint-plugin-prettier-internal-rules/blob/2c414e5b4dec5b4dca39aec9dcf3cd0a0d603f11/index.js#L5
- https://github.com/mysticatea/eslint-plugin-node/blob/f45c6149be7235c0f7422d1179c25726afeecd83/package.json#L28
- https://www.npmjs.com/package/eslint-plugin-self-dir

Reply:
You could hack the module resolution like [node core does](https://github.com/nodejs/node/blob/91d28d8b46cbb9b101ee2c51a32adafb12bc24b7/.eslintrc.js#L23-L34) but i guess it's even worse of a hack 😉.